### DISCUSSION TITLE
Tips for integrating large snATAC-seq datasets

### DISCUSSION THREAD

**Discussion by wenweiliang** [stuart-lab/signac#461] [Q&A] [answered]
Hi, 

Thanks for your inspiring works in Signac development. Just found out a very useful Seurat vignette for `Tips for integrating large datasets` [here](https://satijalab.org/seurat/articles/integration_large_datasets.html). I'm doing exactly the same things for snATAC-seq by following `Merging objects`[this vignette](https://satijalab.org/signac/articles/merging.html). However it was killed several times because it is computationally expensive. Would you mind sharing a few tips on merging large snATAC-seq datasets just like the one on snRNA-seq datasets? Thanks.

**Comment by timoast** [comment, marked as answer]
To reduce the memory requirements, you can remove any assays that are not required. For example, if you initially created a separate Seurat object for each dataset containing non-shared peaks, you could remove that assay and only keep the assay containing shared peaks. You can also filter the peaks set that you use to only contain high-confidence peaks that don't overlap genomic blacklist sites, and this should reduce the number of features in the dataset. You can also try using the current develop-branch version of Signac, which has some improvements in object merging. 

Note that merging datasets is quite different to the Seurat integration methods, so there aren't many different approaches to take when merging datasets that will alter the computational resources needed.
